### ISSUE TITLE
Using Ollama Embedding Model: bge-large-zh-v1 (ollama.py:166)

### ISSUE THREAD

**Issue by suiquantong** [Canner/WrenAI#494]
**Describe the bug**
Using Ollama Embedding Model: bge-large-zh-v1 (ollama.py:166)

**To Reproduce**
run wren-wren-ai-service-1
**Expected behavior**

**Screenshots**

**Container Logs **
You can execute the following command to get the logs of containers and provide them here:
```
docker logs wrenai-wren-ui-1 >& wrenai-wren-ui.log && \
docker logs wrenai-wren-ai-service-1 >& wrenai-wren-ai-service.log && \
docker logs wrenai-wren-engine-1 >& wrenai-wren-engine.log && \
docker logs wrenai-ibis-server-1 >& wrenai-ibis-server.log
```
[wrenai-ibis-server.log](https://github.com/user-attachments/files/16135875/wrenai-ibis-server.log)
[wrenai-wren-ai-service.log](https://github.com/user-attachments/files/16135876/wrenai-wren-ai-service.log)
[wrenai-wren-engine.log](https://github.com/user-attachments/files/16135877/wrenai-wren-engine.log)
[wrenai-wren-ui.log](https://github.com/user-attachments/files/16135879/wrenai-wren-ui.log)

**Desktop (please complete the following information):**
- OS: Linux/ubuntu22.04
[wrenai-ibis-server.log](https://github.com/user-attachments/files/16135875/wrenai-ibis-server.log)
[wrenai-wren-ai-service.log](https://github.com/user-attachments/files/16135876/wrenai-wren-ai-service.log)
[wrenai-wren-engine.log](https://github.com/user-attachments/files/16135877/wrenai-wren-engine.log)
[wrenai-wren-ui.log](https://github.com/user-attachments/files/16135879/wrenai-wren-ui.log)

- Browser chrome

**Wren AI Information**
- Version: 0.7.0
- LLM_PROVIDER= # azure_openai_llm
- GENERATION_MODEL= # glm4-chat

**Additional context**

**Comment by cyyeh** [comment]
@suiquantong could u also share with me the .env.ai file with credential key removed? Thank you!

**Comment by cyyeh** [comment]
@suiquantong  we've released a new version 0.7.1. could you try again using the latest version?
